reactor-netty-core/src/main/java/reactor/netty/channel/MicrometerChannelMetricsHandler.java

```diff
@@ -155,7 +151,10 @@ public void connect(ChannelHandlerContext ctx, SocketAddress remoteAddress,
 			       status = future.isSuccess() ? SUCCESS : ERROR;
 
 			       observation.stop();
-			});
+
+			           ReactorNetty.setChannelContext(ctx.channel(), parentContextView);
+			           parentContextView = null;
+			   });
```

Comment:
nit: tabulation from lines 155,156 are not aligned with the ones above (lines 152,152)

Reply:
fixed, thanks